libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -968,7 +968,7 @@ void IRGeneratorForStatements::endVisit(IndexAccess const& _indexAccess)
 			*_indexAccess.annotation().type,
 			IRLValue::Storage{
 				slot,
-				0
+				variant<std::string, unsigned>(std::in_place_index<1>, 0)
```

Comment:
This looks rather horrible :(
What about just  `unsigned(0)`?

Reply:
Updated PR